php-fpm/Dockerfile-56

```diff
@@ -127,6 +127,10 @@ RUN if [ ${INSTALL_AEROSPIKE_EXTENSION} = true ]; then \
     && rm /tmp/aerospike-client-php.tar.gz \
 ;fi
 
+RUN if [ ${INSTALL_AEROSPIKE_EXTENSION} = false ]; then \
```

Comment:
```diff
-COPY ./aerospike.ini /usr/local/etc/php/conf.d/aerospike.ini
RUN if [ ${INSTALL_AEROSPIKE_EXTENSION} = true ]; then \
+   COPY ./aerospike.ini /usr/local/etc/php/conf.d/aerospike.ini
    # Install the php aerospike extension
    curl -L -o /tmp/aerospike-client-php.tar.gz "https://github.com/luciano-jr/aerospike-client-php/archive/master.tar.gz" \
    && mkdir -p aerospike-client-php \
    && tar -C aerospike-client-php -zxvf /tmp/aerospike-client-php.tar.gz --strip 1 \
    && ( \
        cd aerospike-client-php/src/aerospike \
        && phpize \
        && ./build.sh \
        && make install \
    ) \
    && rm /tmp/aerospike-client-php.tar.gz \
;fi
```

So you don't need to add the following block

```
RUN if [ ${INSTALL_AEROSPIKE_EXTENSION} = false ]; then \
....
```

Does this resolve your problem.